browser/net/brave_proxying_url_loader_factory.cc

```diff
@@ -348,10 +348,11 @@ void BraveProxyingURLLoaderFactory::InProgressRequest::
 
   // TODO(iefremov): Shorten
   if (ctx_->blocked_by != brave::kNotBlocked) {
-    if (ctx_->cancel_request_explicitly) {
-      OnRequestError(network::URLLoaderCompletionStatus(net::ERR_ABORTED));
+    if (!ctx_->ShouldMockRequest()) {
+      OnRequestError(network::URLLoaderCompletionStatus(net::ERR_BLOCKED_BY_CLIENT));
```

Comment:
@bsclifton I just had to change `net::ERR_ABORTED` to `net::ERR_BLOCKED_BY_CLIENT`. The 4 test failures were occurring because the `XMLHttpRequest`s were having their `onabort` handlers triggered instead of `onerror`.

Reply:
Very nice 😄👍